### ISSUE TITLE
Open external links in new tabs

### ISSUE THREAD

**Issue by bfabio** [italia/developers-italia-onboarding#189]
The external links should be opened in a new tab, possibly with `noopener` and `noreferrer`, and checked for the needed accessibility attributes.

**Comment by andreanironi** [comment]
Sorry, just spotted this issue and I'm wondering if the new tab requirement is a good idea. We investigated this on another project some time ago, and decided to step away from it after reading several advices against it, as it seems to be disrupting the mobile experience for visitors. See for instance: https://www.nngroup.com/articles/new-browser-windows-and-tabs/

I fully agree on the rest of the ticket though.

**Comment by bfabio** [comment]
@andreanironi that's actually a really good point :thinking:

**Comment by libremente** [comment]
That's true @andreanironi, but we also have to consider that this website is not really intended to be used in a mobile environment. WDYT @bfabio?

**Comment by bfabio** [comment]
@libremente I think ideally we should try and follow the guideline in that article (which is superbly written, btw, thanks @andreanironi for the link)

> **Summing It Up: What to Do and When**
> 
> For the most part, always open links in the same browser tab or window. [...]

Right now we're kind of forcing a certain flow to the user, which is the flow we think makes sense to us but on a second thought it might not to everyone.

Anyway, I'd defer that to our UX experts :)
